src/index.js

```diff
@@ -58,6 +67,34 @@ if (GlobalConfiguration.END_SESSION_URL) {
   }
 }
 
+// Sentry Config
+Sentry.init({
```

Comment:
Optional: We could add the release variable here to help track versions on the Sentry dashboard. More info here: https://docs.sentry.io/platforms/javascript/guides/react/configuration/options/#release